deepmd/pd/infer/deep_eval.py

```
@@ -0,0 +1,742 @@
+    logits_out = []
+    err_msg = (
+        f"All inputs should be the same format, "
+        f"but found {type(coords)}, {type(cells)}, {type(atom_types)} instead! "
+    )
+    return_tensor = True
+    if isinstance(coords, paddle.Tensor):
+        if cells is not None:
+            assert isinstance(cells, paddle.Tensor), err_msg
+        if spins is not None:
+            assert isinstance(spins, paddle.Tensor), err_msg
+        assert isinstance(atom_types, paddle.Tensor) or isinstance(atom_types, list)
```

Ruff

Multiple `isinstance` calls for `atom_types`, merge into a single call

Merge `isinstance` calls for `atom_types`

(SIM101)